src/config/main-config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const path = require('path');
+const logger = require('morgan');
+const cors = require('cors')
+const bodyParser = require('body-parser');
+
+module.exports = {
+  init(app, express) {
+    app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
I think we'll probably use JSON payloads rather than URL encoded payloads, so we probably don't need this parser (just bodyParser.json()).

Also, although this isn't a huge deal, it's worth noting that the [bodyParser docs](https://github.com/expressjs/body-parser#express-route-specific) recommend using bodyparsers on a a per-route basis rather than an app-level parser.

Reply:
Agreed, I removed this and left `express.json()`